pkgdb/src/buildenv/realise.cc

```diff
@@ -129,6 +130,17 @@ addDirToStore( nix::EvalState &    state,
 
 /* -------------------------------------------------------------------------- */
 
+/**
+ * @brief Merge all components of the environment into a single store path.
+ * @param state Nix state.
+ * @param pkgs List of packages to include in the environment.
+ *             - outputs of packages declared in the environment manifest
+ *             - flox specific packages (activation scripts, profile.d, etc.)
+ * @param references Set of store paths that the environment depends on.
+ * @param originalPackage Map of store paths to the locked package definition
+ *                        that provided them.
+ * @return The combined store path of the environment.
+ */
```

Comment:
suggestion: keep this consistent with the comment in the header
nit: I think in general comments only go in the header